adoc/admin-updates.adoc

```diff
@@ -101,6 +101,69 @@ The control plane consists of these components:
 * cri-o
 =====
 
+=== Applying an update
+
+[NOTE]
+====
+The following steps are temporary and will be integrated into the official product soon
+====
+
+Updates have to be applied on a per node base, starting from the controlplane, down to the worker nodes.
+
+. To prepare the update, `ssh` to each of the nodes and perform the following steps:
+.. disable the `skuba-update.timer`
++
+----
+sudo systemctl disable --now skuba-update.timer
+----
+.. create a temporary lock on the kubernetes component system packages
++
+----
+sudo zypper addlock kubernetes-client kubernetes-common kubernetes-kubelet cri-o
+----
+.. update kubeadm to the newer version
++
+----
+sudo zypper update kubernetes-kubeadm
+----
+.. choose the `break kubernetes-kubeadm-[VERSION] by ignoring some of its dependencies` solution
+.. remove the locks again
++
+----
+sudo zypper removelock kubernetes-client kubernetes-common kubernetes-kubelet cri-o
+----
+. Now that all nodes are prepared, we can use `skuba` to:
+.. upgrade the master nodes
++
+----
+skuba node upgrade apply --target <master-node-ip> --user <user> --sudo
+----
+.. when all master nodes are upgraded, upgrade the worker nodes
++
+----
+skuba node upgrade apply --target <worker-node-ip> --user <user> --sudo
+----
+
+Now all of your cluster nodes are upgraded. You can verify this by running:
+----
+kubectl get nodes -o wide
+----
+
+To enable the automatic OS updates again, `ssh` to each of the nodes and enable the `skuba-update.timer` again:
+
```

Comment:
For some reason, I get this result (notice the extra `+`):

![2019-07-09-160015_1916x1033_scrot](https://user-images.githubusercontent.com/767943/60894359-cf4dfc00-a262-11e9-9a8d-cd6b837ba1be.png)

This only happens in this case. Rendered with the latest `asciidoctor-pdf` binary.